admin_manual/release_notes.rst

```diff
@@ -0,0 +1,266 @@
+================================
+ownCloud |version| Release Notes
+================================
+
+Recommended Setup for Running ownCloud
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ownCloud runs on multiple operating systems and HTTP servers. For best 
+performance, stability, support, and full functionality we recommend:
+
+* Red Hat Enterprise Linux 7
+* MySQL/MariaDB
+* PHP 5.4 +
+* Apache 2.4
+
+Supported Platforms
+^^^^^^^^^^^^^^^^^^^
+
+* Hypervisors: Hyper-V, VMware ESX, Xen, KVM
+* Server: Windows 2008R2; 2012, Linux (Debian 6, RHEL / Centos 6 and 7, Ubuntu 
+  12 and 14, SLES 11, Univention Corporate Server 3.x)
+* Databases: MySQL 5.x; Microsoft SQL Server 2008R2; SQL Server 2012 R2; Oracle 
+  11g
+* Desktop: Windows XP, Windows 7, Mac OS X 10.7+ (64-bit only), Linux (CentOS 
+  6, Ubuntu 10.04 and 11.04+, Fedora 16+, openSUSE 11.4+)
+* Mobile apps: iOS 7+, Android 4+
+* Webserver: IIS (Windows) and Apache2 (Windows and Linux)
+* Web browser: IE8, IE9+, Firefox v14+, Chrome v18+, Safari v5+
+
+PHP 5.3 Support Dropped
+^^^^^^^^^^^^^^^^^^^^^^^
+
+PHP 5.3 is not supported in ownCloud 8, and PHP 5.4 or better is required.
+
+Disable Apache Multiviews
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If Multiviews are enabled in your Apache configuration, this may cause problems 
+with content negotiation, so disable Multiviews by removing it from your Apache 
+configuration. Look for lines like this:: 
+
+ <Directory /var/www/owncloud>
+ Options Indexes FollowSymLinks Multiviews
+ 
+Delete ``Multiviews`` and restart Apache.
+
+.. https://github.com/owncloud/core/issues/9039
+
+ownCloud Does Not Follow Symlinks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ownCloud's file scanner does not follow symlinks, which could lead to 
+infinite loops. To avoid this do not use soft or hard links in your ownCloud 
+data directory.
+
+.. https://github.com/owncloud/core/issues/8976
+
+No Commas in Group Names
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Creating an ownCloud group with a comma in the group name causes ownCloud to 
+treat the group as two groups.
+
+.. https://github.com/owncloud/core/issues/10983
+
+Hebrew File Names Too Large Windows IIS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On Windows IIS Hebrew file names grow to five times their original size after 
```

Comment:
I guess this also happens on Apache on Windows. So it's a limitation of having oC Server running on the MS Windows platform.